packages/web/src/App.tsx

```diff
@@ -81,10 +85,17 @@ const App: React.FC = () => {
   const { switchOpen: switchDrawer } = useDrawer();
   const [label, setLabel] = useState('');
   const { pathname } = useLocation();
+  const { getConversationTitle } = useConversation();
 
   useEffect(() => {
-    setLabel(getLabelByPath(pathname));
-  }, [pathname]);
+    const chatId = extractChatId(pathname);
+
+    if (chatId) {
+      setLabel(getConversationTitle(chatId) || '');
+    } else {
+      setLabel(items.find((i) => i.to === pathname)?.label || '');
+    }
+  }, [pathname, getConversationTitle]);
```

Comment:
以下のエラーが発生しているようです。
Labelを設定するだけの処理なので、useMemoでラベルを返す感じでもいいかなと思いました！
```
App.tsx:94 Warning: Maximum update depth exceeded. This can happen when a component calls setState inside useEffect, but useEffect either doesn't have a dependency array, or one of the dependencies changes on every render.

```

Reply:
修正しました！